index.html

```diff
@@ -930,7 +953,7 @@
         </li>
       </ol>
       <p>
-        The serializer for a <a>PushSubscription</a> invokes the following steps:
+        The <dfn>serializer</dfn> for a <a>PushSubscription</a> invokes the following steps:
```

Comment:
The owning \<section> has been annotated with the `data-dfn-for` attribute, making that redundant. That doesn't work when a section contains multiple identically named attributes, as is the case with the Init dictionaries.